cypress/integration/resources.spec.js

```diff
@@ -30,9 +35,11 @@ describe('resources', () => {
   const assertThatListHasSameItems = () => compareResourceNames({ shouldBeEqual: true });
 
   beforeEach(() => {
+    cy.wait(2000); // eslint-disable-line cypress/no-unnecessary-waiting
```

Comment:
I don't know if this ultimately solves anything but at least it looked like it allowed some failing tests to be more stable when testing in dev.